addons/pos_restaurant/static/src/app/services/pos_store.js

```diff
@@ -36,7 +36,9 @@ patch(PosStore.prototype, {
                 tables: "FloorScreen",
             };
             screen.page = screens[this.config.default_screen];
-            screen.params = {};
+            if (this.config.default_screen === "tables") {
```

Comment:
```suggestion
            if (screen.page === "FloorScreen") {
```
Also can you write a test case for this